Workbooks/UpdateCompliance/QualityUpdates/Quality Updates.workbook

```diff
@@ -0,0 +1,1040 @@
+              "exportedParameters": [
+                {
+                  "fieldName": "TotalDevices",
+                  "parameterName": "_SelectedDevices"
+                },
+                {
+                  "fieldName": "TargetVersion",
+                  "parameterName": "_SelectedTargetVersion",
+                  "parameterType": 1
+                },
+                {
+                  "fieldName": "TargetBuild",
+                  "parameterName": "_SelectedTargetBuild",
+                  "parameterType": 1
+                }
+              ],
+              "showExportToExcel": true,
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "visualization": "table",
+              "gridSettings": {
+                "formatters": [
+                  {
+                    "columnMatch": "TargetVersion",
+                    "formatter": 0,
+                    "formatOptions": {
+                      "aggregation": "Count"
+                    }
+                  },
+                  {
+                    "columnMatch": "TotalProgress",
+                    "formatter": 4,
+                    "formatOptions": {
+                      "palette": "blue",
+                      "customColumnWidthSetting": "150px"
+                    },
+                    "numberFormat": {
+                      "unit": 1,
+                      "options": {
+                        "style": "decimal"
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "Alerts",
+                    "formatter": 18,
+                    "formatOptions": {
+                      "linkTarget": "WorkbookTemplate",
+                      "thresholdsOptions": "icons",
+                      "thresholdsGrid": [
+                        {
+                          "operator": ">",
+                          "thresholdValue": "50",
+                          "representation": "3",
+                          "text": "{0}{1}"
+                        },
+                        {
+                          "operator": "is Empty",
+                          "thresholdValue": "",
+                          "representation": "Blank",
+                          "text": "0"
+                        },
+                        {
+                          "operator": "Default",
+                          "thresholdValue": null,
+                          "representation": "Blank",
+                          "text": "{0}{1}"
+                        }
+                      ],
+                      "workbookContext": {
+                        "componentIdSource": "workbook",
+                        "resourceIdsSource": "workbook",
+                        "templateIdSource": "static",
+                        "templateId": "community-workbooks/UpdateCompliance/AlertsTargetBuild/Alerts for Target build",
```

Comment:
I think this can just be 
`"community-workbooks/UpdateCompliance/AlertsTargetBuild`

but double check and confirm this